SofaKernel/framework/sofa/helper/gl/VideoRecorderFFMPEG.cpp

```diff
@@ -0,0 +1,164 @@
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture, development version     *
+*                (c) 2006-2018 INRIA, USTL, UJF, CNRS, MGH                    *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+#include <sofa/helper/gl/VideoRecorderFFMPEG.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <stdio.h>      //popen
+#include <cstdio>		// sprintf and friends
+#include <sstream>
+
+namespace sofa
+{
+
+namespace helper
+{
+
+namespace gl
+{
+
+VideoRecorderFFMPEG::VideoRecorderFFMPEG()
+    : m_framerate(25)
+    , m_prefix("sofa_video")
+    , m_counter(-1)
```

Comment:
What is it used for ? It is only set, but never read

Reply:
I merged the Thierry branch with all his modifications into my branch so I supposed he took into account this remark.

Reply:
I merged the Thierry branch with all his modifications into my branch so I supposed he took into account this remark.
I send him a review request.

Reply:
As you can see, that was not included. My commit only had fix to allow execution (correct popen flag for linux, quote on path...).